src/plugin.ts

```diff
@@ -558,6 +560,9 @@ export class MongoDBOIDCPluginImpl implements MongoDBOIDCPlugin {
       this.logger.emit('mongodb-oidc-plugin:missing-id-token');
     }
 
+    const refreshTokenId = getRefreshTokenId(tokenSet.refresh_token);
+    const updateId = updateIdCounter++;
```

Comment:
No need to `MAX_SAFE_INTEGER` right?

Reply:
It would only take a few billion years to reach that, so I think we're good here 😄